Makefile

```diff
@@ -51,7 +51,8 @@ check-copyright:
 
 .PHONY: lint
 lint:
-	build/run-lint-check.sh
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.63.4
```

Comment:
is it possible to install the golint to the current project sub dir, so it will affect the global linter?

Reply:
Good point! Updated to put golint into `_output` dir.